src/api/migrations/20190701162027-create-bookmark.js

```diff
@@ -0,0 +1,44 @@
+export default {
+  up: (queryInterface, Sequelize) => {
+    return queryInterface.createTable('bookmarks', {
+      slug: {
```

Comment:
@truestbyheart I think the slug belongs to the article. Maybe `articleSlug` makes more sense here

Reply:
Great suggestion @nkpremices, let me work on it.

Reply:
cool @truestbyheart